pkg/services/ngalert/state/cache.go

```diff
@@ -6,8 +6,10 @@ import (
 	"net/url"
 	"strings"
 	"sync"
+	"time"
 
 	"github.com/grafana/grafana-plugin-sdk-go/data"
+	"github.com/hashicorp/go-multierror"
```

Comment:
What's your plan for this for the future? 

- I see we use `go-multierror` in many other places.
- dskit also has a multierror
- Go 1.20 has native support for this, but we're still at 1.19

At least with this approach is consistent I guess? Do you think it warrants a comment or an issue to follow up on it eventually?

Reply:
Once we're on Go 1.20 I want to use the native support for multiple errors, until then I kept with `go-multierror` for the exact reasons you've listed here!

Reply:
I'll open a GitHub issue for us to move all use of `go-multierror` in Alerting once we update to Go 1.20.

Reply:
https://github.com/grafana/grafana/issues/63686